### ISSUE TITLE
Documentation missing

### ISSUE THREAD

**Issue by Recmo** [sarumont/py-trello#351]
There is no documentation link in the Readme. The link in #14 only contains a copy of the readme and nothing more.

It would be very useful to have a list of all classes and their the functions.

**Comment by arcsector** [comment]
Agreed - there just seems to be needing a regeneration of the docs and then to upload to readthedocs - sucks that this hasn't happened yet - I can build this super easy locally
